### ISSUE TITLE
Won't open if accessibility access is not checked

### ISSUE THREAD

**Issue by gingerbeardman** [lwouis/alt-tab-macos#435]
**Describe the bug**

- Won't open if accessibility access is not checked

**Screenshots / video**

- https://imgur.com/a/q4pIH4n

**Steps to reproduce the bug**

1. System Preferences > Security & Privacy > Accessibility > AltTab is **unchecked**
2. Run AltTab
3. Observe

**Expected**

- AltTab should launch and request access to Accessibility 

**Actual**

- Does not launch

**Console Logs**

```
Jul 17 13:21:01 matt AltTab[17589]: dyld: lazy symbol binding failed: Symbol not found: _SLSRequestScreenCaptureAccess
	  Referenced from: /Applications/AltTab.app/Contents/MacOS/AltTab
	  Expected in: /System/Library/PrivateFrameworks/SkyLight.framework/Versions/A/SkyLight
Jul 17 13:21:01 matt AltTab[17589]: dyld: Symbol not found: _SLSRequestScreenCaptureAccess
	  Referenced from: /Applications/AltTab.app/Contents/MacOS/AltTab
	  Expected in: /System/Library/PrivateFrameworks/SkyLight.framework/Versions/A/SkyLight
```

**Comment by lwouis** [comment]
Oh! I think you're not on macOS 10.15 Catalina right? I didn't know this could be an issue. I disabled the new Catalina-added permission checks on older macOS versions, but I guess the way this private API links doesn't work well with that scenario!

Really sorry about the issue! I'll look into it! It's tricky for me because I don't have a mac with an older macOS handy though

**Comment by gingerbeardman** [comment]
My pleasure! You're correct: I am on Mojave.

**Comment by lwouis** [comment]
Hey @gingerbeardman, could you please tell me if that build here start on your system?
[AltTab.app.zip](https://github.com/lwouis/alt-tab-macos/files/4939373/AltTab.app.zip)

**Comment by lwouis** [comment]
Hey @gingerbeardman, could you please tell me if that build here start on your system?
[AltTab.app.zip](https://github.com/lwouis/alt-tab-macos/files/4939373/AltTab.app.zip)

**Comment by lwouis** [comment]
I released a new version that hopefully fixes the issue on macOS 10.12-14. If it's fixed for you, can you please close the issue? 👍

**Comment by gingerbeardman** [comment]
Perfect! Many thanks.

![Screen shot 2020-07-17 at 20 29 21](https://user-images.githubusercontent.com/49612/87823925-3c2fa500-c86c-11ea-82ae-d01b39f8b4d0.png)
